### PR TITLE
Removing Incorrect parameters

### PR DESCRIPTION
**Pull request by navysingla**
Removing the incorrect parameters that are no longer supported by APIs

### CODE REVIEW

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 1]
```diff
@@ -542,32 +542,16 @@
...
-          }
         }
       }
     },
     "ModernSubscriptionCreationParameters": {
       "description": "The parameters required to create a new subscription.",
-      "required": [
-        "displayName",
-        "billingProfileId",
-        "skuId"
```
Comment (weidongxu-microsoft) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Why "skuId" is dropped from required as well?

**Review by weidongxu-microsoft — COMMENTED** [review]
@navysingla 

Any background? Removing properties is likely a breaking change.

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json (hunk 1 above)
Comment (navysingla) [reply specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: My bad. We want to remove only billingprofileId here

**Comment by navysingla** [comment]
> @navysingla
> Any background? Removing properties is likely a breaking change.

Additional properties are not used by customers and we don't want them to use it too. It was only meant for internal teams. Billing profile Id was a redundant parameter that we no longer support

**Comment by pilor** [comment]
Since this is a preview api-version I wouldn't block this due to breaking changes.  However, please think about the trade-off between potentially inconveniencing early adopters of your API and introducing these changes in a new preview api-version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
@navysingla 

Re-runing CI.

Let me know if you want to go ahead and merge this.

**Comment by weidongxu-microsoft** [comment]
@navysingla

If you do not need this get merged, I will close this PR in days.

**Comment by navysingla** [comment]
Can we merge it? I don't have permission to merge it

**Comment by navysingla** [comment]
> @navysingla
> Re-runing CI.
> Let me know if you want to go ahead and merge this.

Can you please merge it

**Comment by weidongxu-microsoft** [comment]
@navysingla

Merged.

I just need you to confirm the timing. Certain service would wish approved PR not get merged till certain date.

**Pull request merged**